collator/src/mempool/impls/std_impl.rs

```diff
@@ -289,26 +290,27 @@ impl MempoolAdapter for MempoolAdapterStdImpl {
                 "Will start mempool with state update from mc block"
             );
 
-            if let Some((round, time)) = (config_guard.builder.get_genesis())
-                .filter(|(_, time)| *time > new_cx.consensus_info.genesis_millis)
-            {
+            if let Some(genesis) = (config_guard.builder.get_genesis()).filter(|genesis| {
+                genesis.start_round >= new_cx.consensus_info.genesis_round
```

Comment:
In other places comparation was changed from `>=` to `>`. Should it be the same here?